CHANGELOG.md

```diff
@@ -25,6 +25,12 @@ Upgrade guidance:
   IV names via a second `CreateLoopOpts` field.
 * Support for LLVM versions is now limited to LLVM 16 and LLVM 17. Support for
   earlier LLVM versions has been removed.
+* Support for FMA (fused multiply-add) is required. For x86-64, this means only
```

Comment:
We should say this is only for the target accelerator and for  x86-64 this is only for the host target, where the cpu is used as an accelerator. General uses of x86 as a "host" can be any version.

Reply:
Good point, thanks. Edited.